komodo/lint_symlink_config.py

```diff
@@ -0,0 +1,52 @@
+import argparse
+import json
+import os
+import sys
+from pathlib import Path
+from typing import List, Mapping, Set, Union
+
+
+def parse_args():
+    parser = argparse.ArgumentParser(
+        description=("Verify symlink configuration for komodo is valid."),
+    )
+    parser.add_argument(
+        "config",
+        type=Path,
+        help="a json file describing symlink structure",
+    )
+    return parser.parse_args()
+
+
+def _get_root_nodes(link_dict: Mapping[str, Union[Mapping, str]]) -> Set[str]:
```

Comment:
Code duplication with sanity_check.py. Can you use that directly or put it somewhere common?